index.js

```diff
@@ -1,15 +1,27 @@
 'use strict';
 var isFinite = require('is-finite');
 
-module.exports = function (numbers) {
+module.exports = function (numbers, range) {
 	if (!Array.isArray(numbers)) {
 		throw new TypeError('Expected an array');
 	}
 
 	var ticks = ['▁', '▂', '▃', '▄', '▅', '▆', '▇', '█'];
 	var finiteNumbers = numbers.filter(isFinite);
-	var min = Math.min.apply(null, finiteNumbers);
-	var max = Math.max.apply(null, finiteNumbers);
+	var min;
+	var max;
+
+	if (range && range.min) {
```

Comment:
I would go for `opts = opts || {}` in the beginning so you don't have to check every time.